src/amo/components/SearchForm/index.js

```diff
@@ -143,7 +144,10 @@ export class SearchFormBase extends React.Component {
       return;
     }
 
-    if (value.length > SEARCH_TERM_MAX_LENGTH) {
+    if (value.length < SEARCH_TERM_MIN_LENGTH) {
+      log.debug(oneLine`SEARCH_TERM_MIN_LENGTH: ${SEARCH_TERM_MIN_LENGTH} shortage`);
```

Comment:
I don't think we need to log anything here, but if you wanted to this could be:

`log.debug(oneLine`Search term length is too short to dispatch autocomplete action. Minimum length is: ${SEARCH_TERM_MIN_LENGTH}`);`